pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java

```diff
@@ -20,10 +20,14 @@
 
 import java.io.IOException;
 import java.io.OutputStream;
-import java.util.Enumeration;
+import java.util.*;
```

Comment:
@hangc0276 can we avoid wildcard import?

Reply:
ok，i will update in next commit with metric document.